packages/mimikatz.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,56 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+
+  $category = 'Password Attacks'
```

Comment:
lint complains that this also isn't used

Reply:
`$category` needs to be outside of the try/catch block